### ISSUE TITLE
[Small Enhancment] Add an option to specify PDF rotation

### ISSUE THREAD

**Issue by richard-hajek** [juruen/rmapi#151]
Hey,
It would be super cool if one could use `rmapi` to change orientation of notebooks! 
Thanks

**Comment by ddvk** [comment]
a stupid question from my part, but why?

**Comment by richard-hajek** [comment]
I am not sure how many people will use this but I can offer my use-case. 

I use reMarkable to draw on desktop screenshots, e.g. when I am solving a puzzle and I don't want to redraw it I just screenshot it and send it to reMarkable. I overall really like this setup except for one thing. The screenshots are horizontally oriented so that means for each screenshot I have to manually change the orientation. While that is just 2 taps, doing it several dozens times an hour gets tiring fast.

**Comment by richard-hajek** [comment]
I figured out a possible workaround - I can SSH to the reMarkable, figure out the latest uploaded file and change its orientation from within reMarkable metadata files. While I would still appreciate a better solution, feel free to close this issue if you don't think this feature is desirable.

**Comment by ddvk** [comment]
it is not stupid. 

so you just a way to change the metadata.

there is a request to be able to change lastpage

**Comment by richard-hajek** [comment]
Yes, I'd like to change the metadata. Also my workaround is not working so I still would love a proper feature

**Comment by ddvk** [comment]
> Yes, I'd like to change the metadata. Also my workaround is not working so I still would love a proper feature

for the workaround to work you have to restart `xochitl`. also you can make a script to do the whole thing
